inference-engine/ie_bridges/java/tests/IECoreTests.java

```diff
@@ -1,57 +1,62 @@
+import static org.junit.Assert.*;
+import org.junit.Test;
+
 import org.intel.openvino.*;
 
 import java.util.Map;
 import java.util.HashMap;
 
 public class IECoreTests extends IETest {
-    IECore core;
-    String exceptionMessage;
+    IECore core = new IECore();
     
-    @Override
-    protected void setUp() {
-        core = new IECore();
-        exceptionMessage = "";
-    }
-
+    @Test
     public void testInitIECore(){
         assertTrue(core instanceof IECore);
```

Comment:
We don't need this test, ```core``` will always be instance of ```IECore```.

Reply:
Removed